frontend/database/modules/datasets.js

```diff
@@ -642,30 +666,45 @@ const actions = {
       const { status } = err.response;
       message = `STATUS:${status} The labels of the dataset ${datasetName} with task ${datasetTask} could not be saved`;
       typeOfNotification = "error";
-      console.error(message);
+      statusCall = status;
+      if (status === 403) {
+        throw { response: TYPE_OF_FEEDBACK.NOT_ALLOWED_TO_UPDATE_LABELS };
+      }
     } finally {
-      Notification.dispatch("notify", {
-        message,
-        type: typeOfNotification,
-      });
+      statusCall === 403 ||
+        Notification.dispatch("notify", {
+          message,
+          type: typeOfNotification,
+        });
     }
   },
-  async deleteDataset(_, { workspace, name }) {
-    let url = `/datasets/${name}`;
+  async deleteDataset(_, { workspace, name: datasetName }) {
+    let message = "";
+    let typeOfNotification = "";
+    let statusCall = null;
+
+    let url = `/datasets/${datasetName}`;
     url += `?workspace=${workspace}`;
     try {
       await ObservationDataset.api().delete(url, {
-        delete: [workspace, name],
-      });
-      Notification.dispatch("notify", {
-        message: `${name} have been deleted`,
-        type: "success",
-      });
-    } catch (error) {
-      Notification.dispatch("notify", {
-        message: `It is not possible to delete ${name}`,
-        type: "warning",
+        delete: [workspace, datasetName],
       });
+      message = `${datasetName} have been deleted`;
+      typeOfNotification = "success";
+    } catch ({ response }) {
+      let { status } = response;
```

Comment:
Here, all backend errors will be captured, but only for 403 status response a toast will be shown, right? 

Maybe we should show notification errors in other cases (at least something went wrong saving data)